tests/robot-tests/tests/libs/common.robot

```diff
@@ -23,7 +23,20 @@ do this on failure
     capture large screenshot and html
     set selenium timeout  3
 
+custom testid locator strategy
+  [Arguments]  ${browser}  ${test_id}   ${tag}  ${constraints}
+  Log to console  ${tag}
+  Log to console  ${constraints}
+  Log to console  ${test_id}
+  ${element}=   Execute Javascript  let xPathResult = document.evaluate('//*[@data-testid="${test_id}"]', document); if(xPathResult) return xPathResult.iterateNext(); return [];
+  user waits until element is visible  ${element}  30
```

Comment:
This wait can be removed I think - I think the wait would need to be before the element is fetched with `execute javascript`. If we've retrieved the element already, there is no need to wait for it here, I believe